autonomy/deploy/base.py

```diff
@@ -265,14 +274,69 @@ def _try_update_setup_params(
             for param, value in setup_data:
                 setup_param[param] = value
 
-        if not has_multiple_overrides:
-            setup_param = _get_params_dict(override_dict=override)
-            _try_update_setup_params(setup_param=setup_param, setup_data=data)
-            return
+        try:
+            if not has_multiple_overrides:
+                setup_param = cls._get_config_from_json_path(
+                    override_dict=override, json_path=SETUP_PARAM_PATH
+                )
+                _try_update_setup_params(setup_param=setup_param, setup_data=data)
+                return
 
-        for agent_idx in override:
-            setup_param = _get_params_dict(override_dict=override[agent_idx])
-            _try_update_setup_params(setup_param=setup_param, setup_data=data)
+            for agent_idx in override:
+                setup_param = cls._get_config_from_json_path(
+                    override_dict=override[agent_idx], json_path=SETUP_PARAM_PATH
+                )
+                _try_update_setup_params(setup_param=setup_param, setup_data=data)
+        except KeyError:
+            logging.warning(
+                f"Could not update the setup parameter for {skill_id}; "
+                f"Configuration does not contain the json path to the setup parameter"
+            )
+
+    def _try_update_tendermint_params(
+        self,
+        override: Dict,
+        skill_id: PublicId,
+        has_multiple_overrides: bool = False,
+    ) -> None:
+        """Try update the tendermint parameters"""
```

Comment:
Method to update the tendermint params eg. `tendermint_url` and `tendermint_com_url`.

The docker compose build will look like this

```yaml
      - SKILL_HELLO_WORLD_ABCI_MODELS_PARAMS_ARGS_TENDERMINT_URL=http://node0:26657
      - SKILL_HELLO_WORLD_ABCI_MODELS_PARAMS_ARGS_TENDERMINT_COM_URL=http://node0:8080
```

and kubernetes build will look like this

```yaml
        - name: SKILL_HELLO_WORLD_ABCI_MODELS_PARAMS_ARGS_TENDERMINT_URL
          value: http://localhost:26657
        - name: SKILL_HELLO_WORLD_ABCI_MODELS_PARAMS_ARGS_TENDERMINT_COM_URL
          value: http://localhost:8080
```